tests/email/email_parsing_test.php

```diff
@@ -73,7 +73,8 @@ public function setUp()
 		$phpbb_container->set('ext.manager', $extension_manager, phpbb_mock_container_builder::SCOPE_PROTOTYPE);
 
 		$context = new \phpbb\template\context();
-		$phpbb_container->set('template.twig.extensions.collection', array(new \phpbb\template\twig\extension($context, $user)), phpbb_mock_container_builder::SCOPE_PROTOTYPE);
+		$twig_extension = new \phpbb\template\twig\extension($context, $user);
```

Comment:
@rxu yes, just left it as is.
`setParameter` also accepts 2 arguments instead of 3.
But tests pass OK.

Should I fix these issues too?